extensions/wikia/RTE/js/plugins/template/plugin.js

```diff
@@ -537,8 +537,8 @@ RTE.templateEditor = {
 			if (this.placeholder.parent().exists()) {
 				// If placeholder has a parent = it's already inserted in the article
 				// So we are editing, not creating. Thus, replacing the existing article element
-				// With it's edited version
-				this.placeholder.replaceWith(html)
+				// With it's edited version (but keeping the placeholder)
+				this.placeholder.html($(html).children());
 			} else {
 				RTE.tools.insertElement(html);
```

Comment:
does this popup on hover work for this case?

Reply:
Yeah :D